python/ray/tune/tests/test_trial_scheduler.py

```diff
@@ -889,6 +865,47 @@ def testPerturbationResetsLastPerturbTime(self):
         pbt.on_trial_result(runner, trials[3], result(11000, 100))
         self.assertEqual(pbt._num_perturbations, 2)
 
+    def testLogConfig(self):
```

Comment:
I tried to write this in the style of the surrounding tests, but would be great if you can double check especially the tmpdir and file reading code.